lib/tasks/gobierto_budgets/fixtures.rake

```diff
@@ -114,6 +114,8 @@ namespace :gobierto_budgets do
     def import_categories
       puts "== Importing categories =="
       categories = categories_fixtures do |category|
+        category.merge!('kind' => category['kind'] == 'I' ? 'income' : 'expense')
```

Comment:
Makes sense, I have moved the change to the categories.yml, but we still need the mapping, because whe use categories.yml to create the budget lines.